### ISSUE TITLE
Firebase emulates functions and hosting can't be run together

### ISSUE THREAD

**Issue by jefrydco** [firebase/firebase-tools#1253]
### [REQUIRED] Environment info

**firebase-tools:** 6.9.0

**Platform:** Fedora 30

### [REQUIRED] Test case

Work on default project from `firebase init` command

### [REQUIRED] Steps to reproduce

Run `firebase serve --only functions,hosting`

### [REQUIRED] Expected behavior

```
$ firebase serve --only functions,hosting

=== Serving from '/mnt/Data/Projects/Dir/project'...

i  functions: Preparing to emulate functions.
i  hosting: Serving hosting files from: public
✔  hosting: Local server: http://localhost:5000
✔  functions: ssr: http://localhost:5001/<project-id>/us-central1/ssr
```
### [REQUIRED] Actual behavior

```
$ firebase serve --only functions,hosting

✔  functions: ssr: http://localhost:5001/<project-id>/us-central1/ssr
```

**Note:**
It worked on version 6.8.0

**Comment by samtstern** [comment]
This looks like #1242 and a fix is coming in version 6.9.1.  

For now downgrading is the answer.  Thanks for reporting!

**Comment by samtstern** [comment]
Closing as duplicate of #1242

**Comment by samtstern** [comment]
The fix for this issue has been released in `6.9.1`.
